code/modules/mob/living/simple_animal/hostile/human/halloween.dm

```diff
@@ -643,11 +643,11 @@
 				if(!can_mind_interact(H.mind))
 					continue
 				to_chat(H, "<span class = 'warning'>You feel [diceroll>15 ? "incredibly" : ""] disorientated.</span>")
-				H.hallucination += rand(10,20)*diceroll
+				H.hallucination = clamp(rand(10,20)*diceroll, hallucination, 60)
```

Comment:
i miss SECONDS

Reply:
They decrease at a rate of 1 every tick, meaning all the numbers are double in seconds. "60 seconds" is not indicative of "it actually takes 120 seconds for it to disappear"

Reply:
It's impossible to tell at glance what the fuck the magic numbers are, though

Reply:
I could add comments instead